molecule/default/tests/test_default.py

```diff
@@ -12,7 +12,44 @@
 ).get_hosts("all")
 
 
-@pytest.mark.parametrize("x", [True])
-def test_packages(host, x):
-    """Run a dummy test, just to show what one would look like."""
-    assert x
+@pytest.mark.parametrize("f", ["/etc/ssh/sshd_config.d/99-allow-rsakeys.conf"])
+def test_files(host, f):
+    """Verify that all expected files indeed exist."""
+    ff = host.file(f)
+    assert ff.exists
+    assert ff.is_file
+    assert ff.user == "root"
+    assert ff.group == "root"
+    assert ff.mode == 0o644
+
+
+def test_file_content(host):
+    """Verify that all file contains the expected content."""
```

Comment:
```suggestion
    """Verify that all files contain the expected content."""
```

Reply:
Would this instead be the following because we only check a single file? Alternatively something more descriptive than just "the file" would be helpful.

```suggestion
    """Verify that the file contains the expected content."""
```

Reply:
Yup, @mcdonnnj's suggestion makes sense and is probably what @jsf9k intended.

Reply:
Please see commit 9145465.